tools/RAiDER/gnss/processDelayFiles.py

```diff
@@ -429,6 +437,11 @@ def main(
     # estimate residual
     dfc['ZTD_minus_RAiDER'] = dfc['ZTD'] - dfc[raider_delay]
 
+    # estimate noise corrected residual using GNSS sigZTD
```

Comment:
This is not quite right because it assumes that the standard deviation of this column will be the same as the standard deviation of the residuals minus the mean standard deviation of the ZTD, which is not true:
```
Var[X - Y] = Var[X] + Var[Y] - 2Cov(X, Y)
```
Here "X" can be the residuals and "Y" can be the sigZTD. What we want (or at least what we say in the paper) is 
```
\sigma_{model} = \sigma_X - \sigma_Y
```
which is quite different. You can't calculate the difference on a per-observation basis, only for the overall population variances. I'd suggest to leave this part off.